assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.spec.js

```diff
@@ -1,67 +1,172 @@
-/*global bard $controller should $httpBackend crowbarFactory assert $q $rootScope */
+/* global bard $controller should $httpBackend upgradeFactory crowbarFactory
+ assert $q $rootScope ADMIN_UPGRADE_TIMEOUT_INTERVAL */
 describe('Upgrade Flow - Upgrade Administration Server Controller', function () {
     var controller,
-        completedUpgradeResponse = {
-            data: {
-                version: '3.0',
-                addons: [],
-                upgrade: {
-                    upgrading: false,
-                    success: true,
-                    failed: false
+        completedUpgradeResponseData = {
+            current_step: 'admin_upgrade',
+            substep: null,
+            current_node: null,
+            steps: {
+                upgrade_prechecks: {
+                    status: 'pending',
+                    errors: {}
+                },
+                admin_backup: {
+                    status: 'pending',
+                    errors: {}
+                },
+                admin_repo_checks: {
+                    status: 'pending',
```

Comment:
Shouldn't `upgrade_prechecks`, `admin_backup` and `admin_repo_checks` be `passed` instead??